public/locales/cs/translation.json

```diff
@@ -47,11 +47,23 @@
 		"Invalid format, only lower-case letters, numbers, dash and underscore are allowed": "Nesprávný formát, pouze malá písmena, číslice, pomlčka a podtržítko jsou povoleny",
 		"Only lower-case letters, numbers, dash and underscore are allowed": "Pouze malá písmena, číslice, pomlčka a podtržítko jsou povoleny",
 		"Username editing is not allowed": "Není povoleno upravit uživatelské jméno",
-		"Password": "Heslo",
-		"Re-enter Password": "Heslo znovu",
-		"Password can't be empty!": "Heslo nemůže být prázdné!",
-		"Passwords do not match!": "Hesla se musí shodovat!",
-		"Required field": "Povinný údaj"
+		"Required field": "Povinný údaj",
+		"PasswordChange": {
```

Comment:
@byewokko We use 2 level nesting only in some special cases for translations (or we did in the very first implementation of localization - so you may still find it somewhere). I know its possible, but I would advice to create a separate translation key, than to make a nested out of it.